easybuild/tools/options.py

```diff
@@ -568,6 +588,13 @@ def _postprocess_list_avail(self):
             self.log.info(msg)
         else:
             print msg
+
+        # cleanup tmpdir
+        try:
+            shutil.rmtree(self.options.tmpdir)
+        except OSError as err:
+            raise EasyBuildError("Failed to clean up temporary directory %s: %s", self.options.tmpdir, err)
```

Comment:
is it really needed to raise an error here? Can't we just emit a warning and continue?